### PR TITLE
Max timedomain

### PR DESCRIPTION
**Pull request by Maxarreaga**
Adding the time domain functionality to the app

### CODE REVIEW

File: Capstone.pro [hunk 1]
```diff
@@ -1,7 +1,6 @@
-QT += \
-    core gui \
-    multimedia
-
+QT       += core gui
+QT += multimedia
+QT += core gui printsupport multimedia
```
Comment (luckuisha) [inline Capstone.pro]: could you revert back to the format in master? maybe you need to merge from master first

File: QT/mainwindow.cpp [hunk 2]
```diff
@@ -31,8 +30,26 @@ void MainWindow::on_FrequencySlider_valueChanged(int value) {
...
+}
+
+void MainWindow::on_playButton_clicked(bool)
+{
+
+    // Get current cursor position when it exists
+    //int position = 0;
+    //
+    std::cout << "Play Button Pushed" << std::endl;
+    QString file = "C:/Users/The_M/Documents/repos/ECE496/ImperialMarch60.wav";
```
Comment (luckuisha) [inline QT/mainwindow.cpp]: change working directory

Comment (dudublad) [inline QT/mainwindow.cpp]: Pls look into using QDir::currentPath() with the fix I showed

File: QT/timedomain.cpp [hunk 3]
```diff
@@ -0,0 +1,107 @@
+#include "timedomain.h"
```
Comment (luckuisha) [inline QT/timedomain.cpp:1]: we should move the `.cpp` and `.h` files to separate directories

@dudublad

**Review by dudublad — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -32,15 +34,21 @@ Expand **Qt > Qt 6.2.1 > Additional Libraries** and select **Qt multimedia**
...
+Contributors:
 Max
 Kyu Bum Kim (kb.kim@mail.utoronto.ca)
 Eduardo Ortenblad (edu.ortenblad@mail.utoronto.ca)
+
+Acknowledgements:
+
+This code uses STK
+
+This code uses QCustomPlot which is built on the QT
```
Comment (luckuisha) [inline README.md]: This code uses QCustomPlot which is built on QT

File: QT/timedomain.h [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+#include "qcustomplot.h"
+#include <QAudioBuffer>
+#include  <iostream>
+class QAudioDecoder;
+
+class TimeDomain : public QCustomPlot
+{
+    Q_OBJECT
+
```
Comment (luckuisha) [inline QT/timedomain.h:13]: comment what each function does

File: QT/mainwindow.cpp (hunk 2 above)
Comment (Maxarreaga) [reply QT/mainwindow.cpp]: added

File: README.md (hunk 4 above)
Comment (Maxarreaga) [reply README.md]: changed

File: QT/timedomain.h (hunk 5 above)
Comment (Maxarreaga) [reply QT/timedomain.h:13]: Done

File: Capstone.pro (hunk 1 above)
Comment (Maxarreaga) [reply Capstone.pro]: changed to fit the format in master

File: QT/timedomain.cpp (hunk 3 above)
Comment (Maxarreaga) [reply QT/timedomain.cpp:1]: They show up in different files in QT, so imo its fine
image

**Review by luckuisha — APPROVED** [review]

**Review by PGardner2 — APPROVED** [review]

**Pull request merged**